docs/contributing/code-style.md

```diff
@@ -368,9 +369,9 @@ Additional notes:
 
 Our message row DOM elements have a custom attribute `zid` which
 contains the numerical message ID. **Don't access this directly as**
-`x.attr('zid')` ! The result will be a string and comparisons (e.g. with
-`<=`) will give the wrong result, occasionally, just enough to make a
-bug that's impossible to track down.
+`x.attr('zid')` ! The result will be a string and comparisons (e.g., with
+`<=`) will occasionally return the wrong result, just often enough
+to make a bug that's impossible to track down.
 
 You should instead use the `id` function from the `rows` module, as in
 `rows.id(x)`. This returns a number. Even in cases where you do want a
```

Comment:
I think I've not seen someone mess this `zid` business up in years, and so it's possible that we could remove this block.

Reply:
Of course, maybe this bit of documentation is _just that effective_ :) happy to remove if that seems the better route.